test/hybrid.jl

```diff
@@ -54,13 +91,13 @@ end
         # horizontal divergence operator applied to all levels
         hdiv = Operators.Divergence()
         @. dudt = hdiv(u * Geometry.Cartesian1Vector(1.0))
```

Comment:
Do we support this horizontal operator with Cartesian13Vectors like in the test above? It seems like we should be able to apply a spectral divergence to the full field, @simonbyrne, no?

Reply:
Or at least we'd want to advect a `Cartesian3Vector` by a `Cartesian1Vector`?